### PR TITLE
Update dependency management for eap6/tomcat adapters

### PR DESCRIPTION
**Pull request by lkubik**
Update dependency management for EAP 6 and Tomcat adapters. EAP 6.4.6.GA artifacts are publicly available in redhat GA repository.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
         <!-- WildFly -->
         <eap.version>7.0.0.Beta</eap.version>
-        <jboss.as.version>7.2.0.Final</jboss.as.version>
+        <jboss.as.version>7.5.6.Final-redhat-2</jboss.as.version>
```
Comment (stianst) [inline pom.xml]: We want to stick with version in Maven central

File: pom.xml [hunk 2]
```diff
@@ -1208,6 +1208,11 @@
             <id>jboss</id>
             <url>http://repository.jboss.org/nexus/content/groups/public/</url>
         </repository>
+        <!-- for org.jboss.as:jboss-as-parent -->
+        <repository>
```
Comment (stianst) [inline pom.xml]: We don't want to include this repository

File: adapters/spi/tomcat-adapter-spi/pom.xml [hunk 3]
```diff
@@ -29,17 +29,26 @@
 	<artifactId>keycloak-tomcat-adapter-spi</artifactId>
 	<name>Keycloak Tomcat Adapter SPI</name>
     <properties>
-        <!-- <tomcat.version>8.0.14</tomcat.version> -->
-        <!-- <tomcat.version>7.0.52</tomcat.version> -->
         <tomcat.version>6.0.41</tomcat.version>
     </properties>
 	<description />
 
+    <dependencyManagement>
```
Comment (stianst) [inline adapters/spi/tomcat-adapter-spi/pom.xml]: Tomcat adapters should use dependency versions from JBoss AS

File: adapters/saml/tomcat/pom.xml [hunk 4]
```diff
@@ -30,6 +30,18 @@
     <artifactId>keycloak-saml-tomcat-integration-pom</artifactId>
     <packaging>pom</packaging>
 
+    <dependencyManagement>
```
Comment (stianst) [inline adapters/saml/tomcat/pom.xml]: Tomcat adapters should use dependency versions from JBoss AS

File: adapters/oidc/tomcat/pom.xml [hunk 5]
```diff
@@ -30,6 +30,18 @@
     <artifactId>keycloak-tomcat-integration-pom</artifactId>
     <packaging>pom</packaging>
 
+    <dependencyManagement>
```
Comment (stianst) [inline adapters/oidc/tomcat/pom.xml]: Tomcat adapters should use dependency versions from JBoss AS

**Comment by lkubik** [comment]
I've deleted the repo and the property with productized version of JBoss AS.

File: adapters/oidc/tomcat/pom.xml (hunk 5 above)
Comment (lkubik) [inline adapters/oidc/tomcat/pom.xml]: What do you mean by JBoss AS. This dependency managament is from JBoss AS.

Comment (stianst) [inline adapters/oidc/tomcat/pom.xml]: I meant Tomcat adapters should not use dependency versions from JBoss AS 7/EAP. Tomcat is an active project, while JBoss AS7 is long been replaced by WildFly.

Comment (stianst) [inline adapters/oidc/tomcat/pom.xml]: Same comment to everything above ;)

**Comment by stianst** [comment]
Tomcat adapters shouldn't use dependency versions from JBoss AS in community. These adapters are not just for EAP6, but also for Tomcat.

**Comment by patriot1burke** [comment]
We need a PR for 1.9.x branch as well before I can accept this PR.

**Comment by lkubik** [comment]
I've deleted the JBoss AS dependency management in tomcat adapters poms. Do you want me to create also PR for the 1.9.x branch? I have those changes in "my branch" anyway...

**Comment by patriot1burke** [comment]
Why?  We should build against same jars that are used in EAP 6 as we 
need to support an adapter for that.

On 3/29/2016 10:52 AM, Lukas Kubik wrote:
>
> I've deleted the JBoss AS dependency management in tomcat adapters 
> poms. Do you want me to create also PR for the 1.9.x branch? I have 
> those changes in "my branch" anyway...
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/keycloak/keycloak/pull/2348#issuecomment-202936482>
>

-- 
Bill Burke
JBoss, a division of Red Hat
http://bill.burkecentral.com

**Comment by lkubik** [comment]
Are you asking why I removed the dep. management for the tomcat adapters? Stian answered this few comments up. I don't know if it would work in tomcat with EAP deps but IMHO yes.

**Comment by stianst** [comment]
Can we close this PR now and revisit in 2.x when we introduce a "product" profile in the community repo?

**Comment by stianst** [comment]
Closing, let's revisit this when we introduce product profile

**Comment by lkubik** [comment]
OK, I will create a JIRA for this so we don't forget about it.

**Comment by stianst** [comment]
Create one jira for creating product profile and add comment to it. That
way we have a single place to cover everything

**Pull request closed without merging**